tools/analyzer_plugin/test/integration/assists/add_props_test.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+
+import 'package:over_react_analyzer_plugin/src/assist/add_props.dart';
+import 'package:over_react_analyzer_plugin/src/async_plugin_apis/assist.dart';
+import 'package:test/test.dart';
+import 'package:test_reflective_loader/test_reflective_loader.dart';
+
+import '../test_bases/assist_test_base.dart';
+
+void main() {
+  defineReflectiveSuite(() {
+    defineReflectiveTests(AddPropsAssistTest);
+  });
+}
+
+@reflectiveTest
+class AddPropsAssistTest extends AssistTestBase {
+  static int expectedPriority = AddPropsAssistContributor.addPropsKind.priority;
+
+  @override
+  AsyncAssistContributor getContributorUnderTest() =>
+      AddPropsAssistContributor();
+
+  Future<void> test_noPropsAndNoParens() async {
+    var source = newSource('test.dart', '''
```

Comment:
> A helper to load a render method from a path would be really nice, especially for harder cases

_From Luis on the previous PR_

Reply:
Actually I'm not sure I follow, @luisgomez-wk do you have an example of what you mean?

Reply:
I was thinking something like `newSourceFromPath('../playground/foo.dart')`

Thinking about it again though, it seems unnecessary. Especially since I learned you can use a comment to get syntax highlighting in JetBrains, e.g.
```dart
newSource('test.dart', /*language=dart*/ r'''
  import 'package:over_react/over_react.dart';
''');
```

Reply:
Gotcha. Yeah given that the source contents needed for each test suite will differ, too, I think it'd be tough to write a utility for that in the test base class.